test/Api.IntegrationTest/SecretsManager/Controllers/SecretsControllerTests.cs

```diff
@@ -148,11 +148,8 @@ public async Task CreateWithoutProject_RunAsAdmin_Success()
         var (org, _) = await _organizationHelper.Initialize(true, true);
         await LoginAsync(_email);
 
-        var project = await _projectRepository.CreateAsync(new Project { OrganizationId = org.Id, Name = "123" });
```

Comment:
Can you create secrets without projects now? Seems to better align with the tests name now though 👍

Reply:
Admins can now.